config/resource-manager.hcl

```diff
@@ -293,7 +293,7 @@ service "recoveryservices" {
 }
 service "recoveryservicessiterecovery" {
   name      = "RecoveryServicesSiteRecovery"
-  available = ["2022-05-01", "2022-09-10"]
+  available = ["2022-10-01"]
```

Comment:
any chance you could rebase this and delete the following paths:

> ./data/Pandora.Definitions.ResourceManager/RecoveryServicesSiteRecovery/v2022_05_01/
> ./data/Pandora.Definitions.ResourceManager/RecoveryServicesSiteRecovery/v2022_09_10/

Since these API versions don't get removed automatically